### ISSUE TITLE
CE4051 - Need to complete week 3, 4 assignments

### ISSUE THREAD

**Issue by sprince0031** [sprince0031/UL_Masters#2]
- [x] Task 1 - Perceptron
- [x] Task 1 - (Optional) Adaline
- [x] Task 2 - ScikitLearn's Perceptron Algorithm
- [x] Task 3 - Exploration on the "Commonality and differences between artificial neurons and biological neurons".
- [x] Submit all notebooks.

**Comment by sprince0031** [comment]
> * [ ]  Task 1 - Perceptron
> * [ ]  Task 1 - (Optional) Adaline
> * [ ]  Task 2 - ScikitLearn's Perceptron Algorithm
> * [ ]  Task 3 - Exploration on the "Commonality and differences between artificial neurons and biological neurons".
> * [ ]  Submit all notebooks.

### Notes and quick references:
- [Single-Layer Neural Networks and Gradient Descent](https://sebastianraschka.com/Articles/2015_singlelayer_neurons.html)
- [ML FAQ](https://sebastianraschka.com/faq/docs/diff-perceptron-adaline-neuralnet.html)

**Comment by sprince0031** [comment]
Finished the assignment but it was late. I could have completed this well in advance. Should not happen again.
